### DISCUSSION TITLE
Does Electrs/bitcoin on different servers?

### DISCUSSION THREAD

**Discussion by mattdale77** [romanz/electrs#875] [Q&A]
Does Electrs and the bitcoin core it connects to have to be on the same hardware or could they be separate?

If separate, is there a significant disadvantage?

**Comment by romanz** [comment]
It should work in both cases.

If the machine is powerful enough, it's more efficient to run electrs & bitcoind on the same machine - since electrs fetches blocks from bitcoind while indexing & retrieving address history.

**Reply by mattdale77** [reply]
Thanks. 
Good to know.
I am already running them together but I might need to spin up another Electrs on another machine. It's good to know I don't need another bitcoin node
